libpod/runtime_img.go

```diff
@@ -877,8 +867,59 @@ func (r *Runtime) GetHistory(image string) ([]ociv1.History, []types.BlobInfo, s
 }
 
 // ImportImage imports an OCI format image archive into storage as an image
-func (r *Runtime) ImportImage(path string) (*storage.Image, error) {
-	return nil, ErrNotImplemented
+func (r *Runtime) ImportImage(path string, options CopyOptions) error {
+	r.lock.RLock()
+	defer r.lock.RUnlock()
+
+	if !r.valid {
+		return ErrRuntimeStopped
+	}
+
+	file := TarballTransport + ":" + path
+	src, err := alltransports.ParseImageName(file)
+	if err != nil {
+		return errors.Wrapf(err, "error parsing image name %q", path)
+	}
+
+	updater, ok := src.(tarball.ConfigUpdater)
+	if !ok {
+		return errors.Wrapf(err, "unexpected type, a tarball reference should implement tarball.ConfigUpdater")
+	}
+
+	annotations := make(map[string]string)
+	annotations[ociv1.AnnotationDescription] = "test image built"
```

Comment:
Should this be kept in? Looks like a debugging statement

Reply:
removed.